github/ApplicationOAuth.py

```diff
@@ -37,9 +37,11 @@
 
 
 class ApplicationOAuth(NonCompletableGithubObject):
-    """
-    This class is used for identifying and authorizing users for Github Apps.
-    The reference can be found at https://docs.github.com/en/developers/apps/building-github-apps/identifying-and-authorizing-users-for-github-apps
+    """This class is used for identifying and authorizing users for Github Apps.
```

Comment:
Can we keep the first line below `"""`? Is this configurable?

Reply:
I do not think so, as the standard is to have online and then the remaining docs...
Any reason for this special formatting, just personal preference?

Reply:
@EnricoMi ?